docs/Developers/Writing_Documentation.rst

```diff
@@ -28,7 +28,7 @@ You should run this command before trying to build the documentation:
   pip3 install -r docs-requirements.txt
 
 You can test the documentation locally through simply running
-``make html`` in the root directory. This generates
+``make html`` in the docs directory. This generates
```

Comment:
The issue is wrong. Should be `python3 setup.py docs` in root directory

Reply:
So, should I change it to 
``python3 setup.py docs`` in the root directory.